internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -107,29 +104,6 @@ func NewApplicationSnapshotImage(ctx context.Context, component app.SnapshotComp
 		return nil, err
 	}
 
-	// Return an evaluator for each of these
```

Comment:
Removal of this block, also means we can probably remove the `p policy.Policy` parameter from this fucntion.